### ISSUE TITLE
[BUG]: 404 error /api/v0/users/112/resetPassword

### ISSUE THREAD

**Issue by atharva3010** [pcl-labs/meredith-core#332]
```
/api/v0/users/112/resetPassword
```
Response:
```
{
  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.4",
  "title": "Not Found",
  "status": 404,
  "traceId": "00-7df2666278de8e42b373b0a7c0a78f3f-079f41de867edd42-00"
}
```
I am a super admin

**Comment by StrangeWill** [comment]
Ah yeah, didn't update this filter for the tenant changes, done.

**Comment by StrangeWill** [comment]
Fixed in bc4865cae1a568abcbb9fc1b9368d7dc60caee04
